sed/loader/mpes/loader.py

```diff
@@ -847,8 +847,8 @@ def gather_metadata(
             if np.allclose(
                 lens_volts,
                 lens_volts_config,
-                rtol=0.005,
-            ):  # Equal upto 0.5% tolerance
+                rtol=0.02,
```

Comment:
Why is this changing here? Is it just that it is necessary for the testing data?

This is not blocking the PR for me but I just don't understand the change.

Reply:
Somehow the voltages in the test file are somewhat different than our usual settings. Not sure why. I will correct this in the tests.